2003-02-24  Elena Zannoni  <ezannoni@redhat.com>

        * gdb.gdb/selftest.exp: Add matching on specific Red Hat only version
        string.

Index: gdb-6.8/gdb/testsuite/gdb.gdb/selftest.exp
===================================================================
--- gdb-6.8.orig/gdb/testsuite/gdb.gdb/selftest.exp	2008-01-26 14:56:37.000000000 +0100
+++ gdb-6.8/gdb/testsuite/gdb.gdb/selftest.exp	2008-07-14 10:23:50.000000000 +0200
@@ -354,6 +354,9 @@ proc test_with_self { executable } {
 	-re ".\[0-9\]+ = +.+ +0x.*\[0-9.\]+.*$gdb_prompt $" {
 	    pass "printed version with cast"
 	}
+	-re ".\[0-9\]+ = .(Fedora|Red Hat Enterprise Linux) \[\\(\\)0-9.a-z\\-\]+.*$gdb_prompt $" {
+	    pass "printed version Fedora or Red Hat Enterprise Linux only"
+	}
 	-re ".*$gdb_prompt $"	{ fail "printed version" }
 	timeout		{ fail "(timeout) printed version" }
     }
